classroom.rb

```diff
@@ -4,6 +4,7 @@ class Classroom
 
   def initialize(label)
     @label = label
+    @students = []
```

Comment:
- [ ] Kindly remove this line of code and review the comment.

Reply:
@ZeenatLawal I don't understand... so I only want to do is removing the @students line ??